solr/CHANGES.txt

```diff
@@ -177,6 +77,15 @@ Improvements
 
 * SOLR-14933: Ability to add T and P type replica from admin UI (Sayan Das via Eric Pugh)
 
+* SOLR-14223: PKI Auth can bootstrap from existing key files instead of creating new keys on startup (Mike Drob)
+
+* SOLR-11725: Use corrected sample formula for computing stdDev and variance in JSON aggregations
+  (hossman, Munendra S N, yonik)
+
+* SOLR-14387: SolrClient.getById() will escape comma separater within ids (Markus Schuch via Mike Drob)
+
+* SOLR-10814: Add short-name feature to RuleBasedAuthz plugin (Mike Drob, Hrishikesh Gadre)
+
```

Comment:
AFAIK, this shouldn't be in either 8x or 8_7 branches. These are master only changes

Reply:
I have created PR #2033 which fixes changelog for branch_8x. If it looks good then, we can port it to branc_8_7

Reply:
Also, there are entries like below
```
* SOLR-14859: DateRangeField now throws errors when invalid field/fieldType options specified; no longer silently accepts incompatible option values
  (Jason Gerlowski, Chris Hostetter, Munendra S N)
```
This was released in 8.6.3, not sure if it should be again included in 8.7 release

Reply:
Based on our release process, changes entries for released versions should not appear in the changes of unreleased versions, so we should drop this one indeed. See https://cwiki.apache.org/confluence/display/LUCENE/ReleaseTodo#ReleaseTodo-SynchronizeCHANGES.txt.